tests/test_channel_metrics/test_completely_bounded_trace_norm.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from toqito.channels import dephasing
+def test_diamond_norm_quantum_channel():
```

Comment:
We should place in spacing between the functions and between the import statements and the functions. Refer to the other test files to see how this is done.